app/uk/gov/hmrc/calculatepublicpensionadjustment/models/calculation/CalculationRequest.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.calculatepublicpensionadjustment.models.calculation
+
+import play.api.libs.json.{Format, Json, Reads, __}
+import uk.gov.hmrc.calculatepublicpensionadjustment.models.calculation.cppa.CppaTaxYear
+
+case class CalculationRequest(scottishTaxYears: List[String], taxYears: List[CppaTaxYear])
```

Comment:
Should the Scottish tax years be typed, or an int? Is this the tax year ending, or a calendar year?